### PR TITLE
211 Blog for teal and Posit Shiny Contest 2024

### LINKED ISSUE #211: Blog Post:  2024 Posit Shiny Contest submission for teal app
### Blog Post

https://posit.co/blog/announcing-the-2024-shiny-contest/

I'm planning to write a blog post about how teal is collaborating with Posit on the 2024 Shiny Contest. 

In addition to announcing this exciting news to teal and pharmaverse users, I'd like to highlight the `random.cdisc.data` R package and `teal.gallery`, as they will be very useful for creating teal apps using synthetic data.

### PR DESCRIPTION
**Pull request by donyunardi**
Closes #211 

Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [x] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [x] Choose (possibly several) `tag(s)` or categories from the current list: ` c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, propose a new tag to the maintainers! Note: if you use a tag not from this list, the "Check Post Tags" CICD pipeline will error. We occasionally tidy up all `tags` for consistency.
- [x] Add a short description for your blog post in the `description` field at the top of the markdown document.
- [x] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [x] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [x] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by pawelru** [comment]
Thanks Dony for a great post. I have nothing to add. Short & sweet - exactly how it should be.

I'm un-drafting this and asking for further review

File: posts/2024-08-19_teal_submission_.../teal_submission_for__posit__shiny__contest_2024.qmd [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "teal and Posit Shiny Contest 2024"
+author:
+  - name: Dony Unardi
+  - name: Pawel Rucki
+description: "this blog is to announce teal category in Post Shiny Contest 2024 and to inform users on resources to make their teal app for the submission."
+date: "2024-08-19"
```
Comment (StefanThoma) [inline posts/2024-08-19_teal_submission_.../teal_submission_for__posit__shiny__contest_2024.qmd:7]: Please update the date to the final publication date

File: posts/2024-08-19_teal_submission_.../teal_submission_for__posit__shiny__contest_2024.qmd [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Shiny, Technical]
```
Comment (StefanThoma) [inline posts/2024-08-19_teal_submission_.../teal_submission_for__posit__shiny__contest_2024.qmd:10]: I think the Technical tag is not appropriate, please replace with "Community"

File: posts/2024-08-19_teal_submission_.../teal_submission_for__posit__shiny__contest_2024.qmd [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: "teal and Posit Shiny Contest 2024"
+author:
+  - name: Dony Unardi
+  - name: Pawel Rucki
+description: "this blog is to announce teal category in Post Shiny Contest 2024 and to inform users on resources to make their teal app for the submission."
```
Comment (StefanThoma) [inline posts/2024-08-19_teal_submission_.../teal_submission_for__posit__shiny__contest_2024.qmd:6]:
```suggestion
description: "Introducing the {teal} category in the Post Shiny Contest 2024 and informing users on resources to make their teal app for the submission."
```

**Review by StefanThoma — CHANGES_REQUESTED** [review]
Happy to hear about this new category! :)

**Review by bms63 — APPROVED** [review]
Looks great to me.

**Pull request merged**